proposal-selection.md

```diff
@@ -0,0 +1,108 @@
+*   You will need to have won the trust of this community in order to have them back your proposal.
+*   You will need a [mentor](roles.html) before you get an application approved.
+*   You should fit with the [eligibility criteria of Season of Docs:](https://developers.google.com/season-of-docs/docs/faq#what_are_the_eligibility_requirements_for_participation)
+    *   You must be at least 18 years of age when you register.
+    *   You must be eligible to work in the country you will reside in during the program.
+    *   You must reside in a country that is not currently embargoed by the United States. See the [program rules](https://developers.google.com/season-of-docs/terms/program-rules) and the [Sanctions Programs and Country Information](https://www.treasury.gov/resource-center/sanctions/Programs/Pages/Programs.aspx) from the US Treasury.
+*   We support diversity and encourage applications and involvement from underrepresented communities.
+
+## Implementation process
+
+A variant on the following process should be considered for major initiatives.
+
+**Test the waters:**
+
+*   Float an idea in our community forums.
+*   Look for feedback, interest and engagement. 
+*   If you attract significant community interest and support, and the required effort is more than is likely to be addressed by volunteers, you might consider developing a proposal for sponsored work.
+
+**Prepare proposal:**
+
+*   Develop a proposal in conjunction with the community. Make sure it addresses our project priorities.
+*   Find a trusted mentor from our community who is willing to mentor you.
+*   Share proposal.
+
+**Review proposals:**
+
+*   Community review and discuss proposals.
+*   Community decides if the proposal is worth backing, and locks in if it is.
+
+**Implementation phase:**
+
+*   The worker works according to the proposed plan.
+*   The worker checks in weekly with mentors and other collaborators.
+*   Worker provides a weekly status. This can be brief, just a few lines.
+
+**Wrap up:**
+
+*   Write a report, hand over documents and present to the community.
+*   Send an announcement to the main mailing and slack #announcements channel.
+*   Arrange for a tweet and/or blog post to be created announcing the project completion, linking to the finished work.
+
+## Payment process
+
+*   Where applicable, guidance on budget will typically be based on the [stipend amounts](https://developers.google.com/season-of-docs/docs/tech-writer-stipends) used by Season of Docs (20-30 hours per week x 3 months, or equivalent effort spread over 6 months). This will be referenced from the Request for Proposal.
+*   Payment milestones will typically be every 3 months.
+*   Payments should be correlated to milestones as github tasks. 
+*   When a milestone is reached:
+    *   The implementor should comment on the milestone task, asking the mentor for review. 
```

Comment:
lol @ "implementor", I keep reading it something like "dementor" :joy:  Perhaps it would be a good idea to define these roles somewhere? I'm not 100% certain if an implementor is the person doing the work (mentee?) or some other person in the community.

Reply:
If this is quick to fix then I'm 👍 otherwise leave it for the next iteration of this document IMHO.